content/en/docs/security/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Security Documentation"
+linkTitle: "Security"
+description: "Flux Security documentation."
+---
+
+<!-- For doc writers: Step-by-step security instructions should live on the appropriate documentation pages. To fulfil our promise to end users, we should briefly outline the context here, and link to the more detailed instruction pages from each relevant part of this outline. -->
+
+{{% alert color="info" title="✍️⏳ Work in progress" %}}
+This document is a work in progress.
+Please follow [this GitHub issue](https://github.com/fluxcd/website/issues/598) to stay updated on goals and progress as we complete this page.
+{{% /alert %}}
+
+## Introduction
+
+Flux has a multi-component design, and integrates with many other systems.
+
+This document outlines an overview of security considerations for Flux components, project processes, artifacts, as well as Flux configurable options and what they enable for both Kubernetes cluster and external system security.
+
+See our [security processes document](/security) for vulnerability reporting, handling, and disclosure of information for the Flux project and community.
+
+## Controller permissions
+
+While Flux integrates with other systems it is built on Kubernetes core controller-runtime and properly adheres to Kubernetes security model including RBAC.
+
+<!-- See [Flux RBAC manifests](https://github.com/fluxcd/flux2/tree/main/manifests/rbac)
+- Scott: spell out here in high-level human-readable terms -->
+
+Flux installs a set of [RBAC manifests](https://github.com/fluxcd/flux2/tree/main/manifests/rbac).
+These include:
+
+1. A `crd-controller` `ClusterRole`, which:
+    - Has full access to all the Custom Resource Definitions defined by Flux controllers
+    - Can get, list, and watch namespaces and secrets
+    - Can get, list, watch, create, patch, and delete configmaps and their status
+    - Can get, list, watch, create, patch, and delete coordination.k8s.io leases
+2. A `crd-controller` `ClusterRoleBinding`:
+    - References `crd-controller` `ClusterRole` above
+    - Bound to a service accounts for every Flux controller
+3. A `cluster-reconciler` `ClusterRoleBinding`:
+    - References `cluster-admin` `ClusterRole`
+    - Bound to service accounts for only `kustomize-controller` and `helm-controller`
+
+Flux uses these two `ClusterRoleBinding` strategies in order to follow the principle of least privilege, and to allow for clear access separation using tools purpose built for policy enforcement (OPA, Kyverno, admission controllers).
```

Comment:
The permissions needed by the controllers, and the permissions granted by the implementation as described above, are different. For example, the controllers don't _all_ need full access to _all_ of the API -- so it doesn't adhere to the principle of least privilege, for the sake of convenience.

Reply:
@squaremo Thanks, removing for now. 

However, do we have a list of these minimal permissions anywhere so that users who want to follow this principle will know the minimum privilege they require – even if that's not the default on bootstrap? I could see this being a potentially valuable option for some users. WDYT?